uint8 motion_type
float32[3] direction
float32 swing_height
uint8 leg_type
uint32 duration